python/cudf/cudf/tests/conftest.py

```diff
@@ -1,10 +1,20 @@
 import pathlib
 
+import pandas as pd
 import pytest
+from packaging.version import parse
 
 import rmm  # noqa: F401
 
 
 @pytest.fixture(scope="session")
 def datadir():
     return pathlib.Path(__file__).parent / "data"
+
+
+@pytest.fixture(scope="session")
+def pd_version_fail():
+    if parse("1.0") <= parse(pd.__version__) < parse("1.1"):
+        return True
+    else:
+        return False
```

Comment:
Instead of defining a session fixture like this we should make a decorator that can be used to tag tests. Additionally we should make this a `pandas_1.0_fail` or something along those lines, and then when Pandas 1.2 comes around we could additionally have a `pandas_1.1_fail` and tag things accordingly.